examples/controlnet/train_controlnet_flax.py

```diff
@@ -50,6 +50,9 @@
 from diffusers.utils import check_min_version, is_wandb_available
 
 
+LARGE_ENOUGH_NUMBER = 100
+PngImagePlugin.MAX_TEXT_CHUNK = LARGE_ENOUGH_NUMBER * (1024**2)
```

Comment:
Maybe add a comment for the users to know why this needs to be set?

Reply:
What exactly does this change do here?

Reply:
added a comment there - to prevent an error I would get when working with coyo700m "Decompressed Data Too Large"